src/app/components/Game/CardHints.tsx

```diff
@@ -55,7 +56,7 @@ export function CardHints({ card }: CardHintsProps) {
                   <FormattedMessage
                     id="hintCardGroup"
                     values={{
-                      cardGroup: card.cardGroup,
+                      cardGroup: card.type,
```

Comment:
Could you rename `cardGroup` to `cardType`? It needs to be updated in the type declaration and `HINTS` array as well (in this file).

The messages also have to be updated (updating the `hintCardGroup` key to `hintCardType`). Ideally, the type should be translated as well.

Here's for the english translation:

```diff
- hintCardGroup: "This is a {cardGroup} card",
+ hintCardType: "This is a {type, select, unit {unit} special {special} artifact {artifact}} card",
```

And french:

```diff
- hintCardGroup: "C'est une carte {cardGroup}",
+ hintCardType: "C'est une carte {type, select, unit {unité} special {spéciale} artifact {artefact}}",
```